### ISSUE TITLE
[macos] Code signing should not be enabled for the example application

### ISSUE THREAD

**Issue by badboy-tian** [google/flutter-desktop-embedding#178]
Command CodeSign failed with a nonzero exit code

**Comment by stuartmorgan** [comment]
There's not enough information here for me to know what the issue is. What specifically were you trying to do when you encountered that error?

The example hasn't been configured for codesigning, so if you were trying to build a signed version of the example it wouldn't be surprising that it wouldn't work without modification. If that is what you were doing, what was your use case?

**Comment by badboy-tian** [comment]
@stuartmorgan thank you for reply. 
![image](https://user-images.githubusercontent.com/2835129/50330437-2a720880-0536-11e9-842e-38e8a4d8d7ce.png)
///////
![image](https://user-images.githubusercontent.com/2835129/50330459-3d84d880-0536-11e9-80f7-301ed1f646c1.png)

**Comment by stuartmorgan** [comment]
Does disabling `Automatically manage signing` fix the issue?

There's no reason I can think of to sign the example.

**Comment by stuartmorgan** [comment]
Re-opening; if that is indeed the issue, then this will hit other people as well, and the project should have that option disabled in the repository.

**Comment by stuartmorgan** [comment]
Hopefully with automatic management disabled this won't hit other people. Thanks for the report!

**Comment by GreenAppers** [comment]
I have an app derived from flutter-desktop-embedding that I want to sign.

It's worked up until this point.  Only after merging the last 8 commits did it start failing.

It's trying to codesign an empty App.framework/../Versions/A directory.

Seems linking App.framework was added here https://github.com/GreenAppers/cruzall-desktop/commit/91283c7477aa532dc4bdb1f27cddc03050a1abc7#diff-066b7547e8b4c8c1dd24c789863d132f

Any suggestions for how to get it signing again?  It was just working a few days ago:  https://github.com/GreenAppers/cruzall-macos

Thanks

**Comment by stuartmorgan** [comment]
Please file a new issue; this bug was about disabling the code-signing settings, which is not what your issue is about.
